cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -431,12 +431,19 @@ func ExecCommandInContainer(cs *testclient.ClientSet, pod corev1.Pod, containerN
 		return buf, fmt.Errorf("cannot create SPDY executor for req %s: %w", req.URL().String(), err)
 	}
 
+	var errorBuf bytes.Buffer
```

Comment:
just a small nit can you please allocation the two buffers together at the top of the function please